sensorlib/src/CarlaCDASimAPI.py

```diff
@@ -108,6 +109,16 @@ def create_simulated_semantic_lidar_sensor(self, simulated_sensor_config, carla_
         parent_actor = CarlaUtils.get_actor(self.__carla_world, parent_actor_id)
         carla_sensor = self.__carla_world.spawn_actor(sensor_bp, sensor_transform, attach_to=parent_actor)
 
+
+        dummy_veh_spawn = carla.Transform(
```

Comment:
What is the purpose of creating this dummy vehicle as part of the library? Is this just a line for testing?

Reply:
removed this as it was only for testing